addon/components/mdc-checkbox.js

```diff
@@ -82,26 +80,32 @@ export default Component.extend(MDCComponent, SupportsBubblesFalse, {
    */
   createFoundation() {
     return new MDCCheckboxFoundation({
-      addClass: className => next(() => addClass(className, this)),
-      removeClass: className => next(() => removeClass(className, this)),
-      registerAnimationEndHandler: handler =>
+      addClass: (className) => next(() => addClass(className, this)),
+      removeClass: (className) => next(() => removeClass(className, this)),
+      registerAnimationEndHandler: (handler) =>
         getElementProperty(this, 'addEventListener', () => null)(ANIM_END_EVENT_NAME, handler),
-      deregisterAnimationEndHandler: handler =>
+      deregisterAnimationEndHandler: (handler) =>
         getElementProperty(this, 'removeEventListener', () => null)(ANIM_END_EVENT_NAME, handler),
-      registerChangeHandler: handler => run(() => get(this, 'changeHandlers').addObject(handler)),
-      deregisterChangeHandler: handler => run(() => get(this, 'changeHandlers').removeObject(handler)),
+      registerChangeHandler: (handler) => run(() => get(this, 'changeHandlers').addObject(handler)),
+      deregisterChangeHandler: (handler) => run(() => get(this, 'changeHandlers').removeObject(handler)),
```

Comment:
Is this really how the linter prefers functions formatted now? Strange, the move away from this in the past seemed fine to me.

Reply:
This is one of the prettier 2.x changes.